ddtrace/constants.py

```diff
@@ -1,23 +1,8 @@
-FILTERS_KEY = "FILTERS"
-SAMPLE_RATE_METRIC_KEY = "_sample_rate"
```

Comment:
We should import these constants back so they don't break existing functionality. Might also be worth commenting the deprecated constants